packages/server-nodejs/src/models/HealthDto.ts

```diff
@@ -1,11 +1,19 @@
 
-import { IsBoolean } from 'class-validator';
+import { z } from 'zod';
+
+export const schema = z.object({
```

Comment:
.object (and others) should start at a new line

Reply:
Removed the whole object. It was not used